ui/lib/core/addon/components/search-select-with-modal.js

```diff
@@ -0,0 +1,217 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { singularize } from 'ember-inflector';
+import { resolve } from 'rsvp';
+import { filterOptions, defaultMatcher } from 'ember-power-select/utils/group-utils';
+
+/**
+ * @module SearchSelectWithModal
+ * The `SearchSelectWithModal` is an implementation of the [ember-power-select](https://github.com/cibernox/ember-power-select) used for form elements where options come dynamically from the API. It can only accept a single model.
```

Comment:
> The `SearchSelectWithModal` is an implementation of the [ember-power-select](https://github.com/cibernox/ember-power-select)
I think this is no longer true since Jordan added the copied methods. Maybe we should say inspired by instead

Reply:
I thought it still was because the hbs still renders the `<PowerSelect>` component?